org.gtk.Gtk3theme.Matcha-dark-sea.json

```diff
@@ -0,0 +1,47 @@
+{
+    "id": "org.gtk.Gtk3theme.Matcha-dark-sea",
+    "branch": "3.22",
+    "runtime": "org.freedesktop.Platform",
+    "build-extension": true,
+    "sdk": "org.freedesktop.Sdk",
+    "runtime-version": "19.08",
```

Comment:
```suggestion
    "runtime-version": "20.08",
```

Reply:
I'm also bumping to 20.08 all the others